scripts/snakemake.sh

```diff
@@ -1 +1 @@
-snakemake -j $(grep -c processor /proc/cpuinfo) --use-singularity --singularity-args "-B $(pwd):/opt/openproblems" all
+snakemake -j 1 --use-singularity --singularity-prefix /tmp/.snakemake --singularity-args "-B $(dirname $(pwd)):/opt/openproblems" all
```

Comment:
Do you need to use -B here? I was a bit confused about that as in docker mounting volumes is done via `-v` or `--mount`... is that singularity syntax?